chapters/request_size.rst

```diff
@@ -0,0 +1,133 @@
+=================
+Request Size
+=================
+
+``request_size`` is the parameter that we could set for the Flow.
+It defines the number of data sent into parallel ``Peas``.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Request Size
+------------
+Jina defines the ``request_size`` as the parameter at client side. By setting the ``request_size`` in Flow's API
+we could define the number of data sent to each ``Peas``.
```

Comment:
we could define the `number` of Documents contained in every `Request`.

Reply:
✔️